tests/bud.bats

```diff
@@ -262,3 +262,25 @@ load helpers
   [ "$status" -eq 0 ]
   [ "$output" = "" ]
 }
+
+@test "bud-shell" {
+  target=alpine-image
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/shell
+  run buildah --debug=false inspect --type=image --format '{{printf "%q" .Docker.Config.Shell}}' ${target}
+  echo $output
+  [ "$status" -eq 0 ]
+  [ "$output" = '["/bin/sh" "-c"]' ]
+  ctr=$(buildah from ${target})
+  run buildah --debug=false config --shell "/bin/bash \-c" ${ctr}
```

Comment:
Does this test succeed without the `\` here?

Reply:
Yes it does.  The urvave only gets confused when it is "-c", it does not complain with "bash -c"

Reply:
Fixed